.github/workflows/windows-selfhosted.yml

```diff
@@ -47,6 +47,10 @@ jobs:
       
     - name: test
       run: |
+        $ErrorActionPreference = "SilentlyContinue"
+        where.exe spirv-dis
+        spirv-dis --version
+        $env:SLANG_RUN_SPIRV_VALIDATION='1'
```

Comment:
I don't think we need this change anymore.
The file "windows-selfhosted.yml" is delete at ToT.
The same functionality is implemented in ci.yml, which sets SLANG_RUN_SPIRV_VALIDATION correctly.

Reply:
Ok, I can remove this file from this change.